.github/workflows/checks.yml

```
@@ -206,3 +206,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           make license-check
+        if: env.LICENSE_CHECK == 'true'
```

By default, license checks do not run.
To activate them, a `LICENSE_CHECK` variable must be configured at the repository level.
This will allow to hot disable them in case another issue is discovered, and thus avoid the need to rush a fix.

---

Alternatively, I can reverse the condition to enable the license checks by default, allowing to disable it in case of issues:
```suggestion
        if: env.LICENSE_CHECK == 'false'
```